gazelle/BUILD.bazel

```diff
@@ -0,0 +1,66 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+load("@bazel_gazelle//:def.bzl", "gazelle_binary")
+
+# Don't have Gazelle generate BUILD files in the testdata directory. It is test
+# data and therefore has strange edge cases that may/may not work.
+# gazelle:exclude testdata
+
+go_library(
+    name = "go_default_library",
```

Comment:
Should this be named something else, or is "go_default_library" a magic identifier?

Reply:
`go_default_library` is a magic identifier defined by [rules_go](https://github.com/bazelbuild/rules_go#what-s-up-with-the-go-default-library-name)